src/schema/sources.ts

```diff
@@ -347,13 +374,74 @@ const sourceToGQL = (source: Source): GQLSource => ({
 });
 
 export enum SourcePermissions {
-  View,
-  Post,
-  Leave,
-  Delete,
-  Edit,
+  View = 'view',
+  Post = 'post',
+  PostDelete = 'post_delete',
+  RemoveMember = 'remove_member',
```

Comment:
Do you want to evaluate mods as well? 
Or is the proposed FE check for owners enough?